src/App.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
-import NavBar from "./Components/NavBar";
-import EmployeeList from "./Components/EmployeeList";
+import NavBar from "./components/NavBar";
+import EmployeeList from "./components/EmployeeList";
```

Comment:
This should be capital C to match your folder. Once it's fixed your code runs as it should.